src/desktop/space/mod.rs

```diff
@@ -538,55 +379,92 @@ pub enum OutputError {
     Unmapped,
 }
 
-render_elements! {
+impl<E: IsAlive> IsAlive for InnerElement<E> {
+    fn alive(&self) -> bool {
+        self.element.alive()
+    }
+}
+
+impl<E: SpaceElement> SpaceElement for InnerElement<E> {
+    fn geometry(&self) -> Rectangle<i32, Logical> {
+        let mut geo = self.element.geometry();
+        geo.loc += self.location;
+        geo
+    }
+
+    fn bbox(&self) -> Rectangle<i32, Logical> {
+        let mut bbox = self.element.bbox();
+        bbox.loc += self.location;
+        bbox
+    }
```

Comment:
This is not working as intended and will result in wrong locations of the elements in the space. The location on screen should be location - geometry.loc iirc.
So maybe we can leave `geometry` as is and only ofset the `bbox`, then in the space during collecting the render element do something like `e.bbox().loc - e.geometry().loc`.